package.json

```diff
@@ -24,9 +24,10 @@
   "scripts": {
     "cz": "git-cz",
     "clean": "rimraf ./dist",
-    "build": "yarn clean && yarn build:umd && yarn build:es",
+    "build": "yarn clean && yarn build:umd && yarn build:es && yarn build:es-package",
     "build:umd": "cross-env NODE_ENV=es rollup -c",
     "build:es": "cross-env BABEL_ENV=es babel ./src --out-dir ./dist/es --ignore '**/*.spec.js'",
+    "build:es-package": "echo '{\"type\":\"module\"}' > dist/es/package.json",
```

Comment:
There's already a root package.json, why would this be necessary? There's some work in progress which should help with ESM: https://github.com/react-dropzone/react-dropzone/pull/1386.

Reply:
@targos I wasn't aware that nesting a `package.json` the way you did will actually work the way you mention. Is this a bundler specific spec?

Reply:
It's not specific to bundlers. The spec comes from Node.js: https://nodejs.org/docs/latest/api/packages.html#type

Reply:
This PR fixes `import('react-dropzone')` in Node.js context.

Reply:
Hmm, so according to:
> import statements of .js files are treated as ES modules if the nearest parent package.json contains "type": "module".

It would mean that the nested `package.json` specifying `"type": "module"` is what makes node interpret the dist/es as ESM.

Thanks for making it more clear as I wasn't aware that nested manifest files would result in such a behaviour.